Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -640,6 +640,12 @@ void MediaPlayerPrivateGStreamer::setRate(float rate)
             m_playbackRatePausedState = PlaybackRatePausedState::RatePaused;
             updateStates();
         }
+        if (m_currentState == GST_STATE_PLAYING && m_playbackRatePausedState != PlaybackRatePausedState::RatePaused) {
```

Comment:
I think we should also add here a `!m_playbackRatePausedState` check. Otherwise the pause might happen unconditionally.
I've already added the extra condition in the patch being submitted upstream. No need to actually update the PR here.

Reply:
Ok, Thanks